draft-ietf-moq-transport.md

```diff
@@ -442,6 +442,29 @@ constrain the information in these fields, for example by restricting them to
 UTF-8. Any specification that does needs to specify the canonicalization into
 the bytes in the Track Namespace or Track Name such that exact comparison works.
 
+### Subscription Identifiers
+
+To optimize wire efficiency, some MoQT messages refer to a track by a numeric
+identifier, rather than the Full Track Name. When a subscriber makes a SUBSCRIBE
+or FETCH request for a track, it associates a Subscriber-Chosen Subscription ID
+(SSID) with the request. SSIDs are scoped to a single MoQT session.  They are
+even; they start at 0 and increase by 2 and have a maxmium value of 2^62-2.  The
+publisher limits the number of requests a subscriber can make using the
+MAX_SUBSCRIPTION_ID SETUP parameter or MAX_SUBSCRIPTION_ID control message.
+
+MoQT also allows the publisher to choose an identifier for each track, called
+the Publisher-Chosen Subscription ID (PSID). PSIDs are odd and have a maximum
+value of 2^61-1. The same identifier MUST NOT be assigned multiple tracks in the
+same session.  PSIDs are required when the publisher initiates the subscription
+(TODO: delete this; PUBLISH message is forthcoming) and are optional when the
+subscriber initiates it.
+
+When a publisher sends Subgroups or Datagrams, these can contain either a
+Subscriber-Chosen or Publisher-Chosen Subscription ID, determined by the LSB of
```

Comment:
Can you spell out least significant bit here.  I suspect most people would figure it out, but I'd rather use the long form once before using the abbreviation.